Bonsai.Miniscope/UCLAMiniscope.cs

```diff
@@ -30,6 +30,10 @@ public class UCLAMiniscope : Source<IplImage>
         [Description("The sensor gain.")]
         public double SensorGain { get; set; } = 16;
 
+        private double lastLEDBrightness;
```

Comment:
Would be best if the change was introduced via a Setter. This would obviate the need for all the difference checking below. Issue is that we need the `capture` object since the setter would need to side effect onto hardware. I suppose we could do an `if (capture != null)` before the side effecting part. Still seems ugly.